src/astro_image_display_api/image_viewer_logic.py

```diff
@@ -685,7 +733,10 @@ def set_viewport(
     set_viewport.__doc__ = ImageViewerInterface.set_viewport.__doc__
 
     def get_viewport(
-        self, sky_or_pixel: str | None = None, image_label: str | None = None
+        self,
+        sky_or_pixel: str | None = None,
+        image_label: str | None = None,
+        **kwargs,  # noqa: ARG002
```

Comment:
That said, maybe we do want `**kwargs` here since viewer in Jdaviz has reference name as such, that might not apply to other backends.